webpack.config.js

```diff
@@ -1,10 +1,12 @@
+/* eslint-disable lodash/prefer-lodash-typecheck */
 const fs = require('fs');
 const path = require('path');
 const webpack = require('webpack');
 const TerserPlugin = require('terser-webpack-plugin');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
+const SriWebpackPlugin = require('webpack-subresource-integrity');
```

Comment:
What is the use of this new package?

Reply:
It's add SHA-512 (or others SHA versions)  to the link & script HTML elements